app/models/manageiq/providers/lenovo/physical_infra_manager/physical_chassis.rb

```diff
@@ -0,0 +1,28 @@
+module ManageIQ::Providers
+  class Lenovo::PhysicalInfraManager::PhysicalChassis < ::PhysicalChassis
+    delegate :product_name,
```

Comment:
I'm wondering if this `delegate` strategy is good here, seems like it is a good thing when you want to support some code that already access this properties like `chassis.product_name`, for example, which is not the case once it is a new component. What do you think @agrare , is there some pattern about this strategy?